keyboards/romac/info.json

```
@@ -0,0 +1,13 @@
+{
+  "keyboard_name": "RoMac",
+  "url": "",
+  "maintainer": "qmk",
```

Ideally, your GitHub username should go here.

```suggestion
  "maintainer": "TheRoyalSweatshirt",
```